content/en/user-guide/aws/autoscaling/index.md

```diff
@@ -0,0 +1,129 @@
+        "LaunchTemplateName": "my-template-for-auto-scaling",
+        "CreateTime": "2024-07-12T07:59:08+00:00",
+        "CreatedBy": "arn:aws:iam::000000000000:root",
+        "DefaultVersionNumber": 1,
+        "LatestVersionNumber": 1,
+        "Tags": []
+    }
+}
+```
+
+### Create an Auto Scaling group
+
+Before creating an Auto Scaling group, you need to fetch the subnet ID. Run the following command to describe the subnets:
+
+{{< command >}}
+$ awslocal ec2 describe-subnets
+{{< /command >}}
+
+Copy the subnet ID from the output and use it to create the Auto Scaling group. Run the following command to create an Auto Scaling group using the [`CreateAutoScalingGroup`](https://docs.aws.amazon.com/autoscaling/ec2/APIReference/API_CreateAutoScalingGroup.html) API:
+
+{{< command >}}
+$ awslocal autoscaling create-auto-scaling-group \
+    --auto-scaling-group-name my-asg \
+    --launch-template LaunchTemplateId=lt-5ccdf1a84f178ba44 \
+    --min-size 1 \
+    --max-size 5 \
+    --vpc-zone-identifier 'subnet-d4d16268'
+{{< /command >}}
+
+### Describe the Auto Scaling group
+
+You can describe the Auto Scaling group using the [`DescribeAutoScalingGroups`](https://docs.aws.amazon.com/autoscaling/ec2/APIReference/API_DescribeAutoScalingGroups.html) API. Run the following command to describe the Auto Scaling group:
+
+{{< command >}}
+$ awslocal autoscaling describe-auto-scaling-groups
+{{< /command >}}
+
+The following output is displayed:
+
+```bash
+{
+    "AutoScalingGroups": [
+        {
+            "AutoScalingGroupName": "my-asg",
+            "AutoScalingGroupARN": "arn:aws:autoscaling:us-east-1:000000000000:autoScalingGroup:74b4ffac-4588-4a7c-86b1-9fa992f49c8e:autoScalingGroupName/my-asg",
+            "LaunchTemplate": {
+                "LaunchTemplateId": "lt-5ccdf1a84f178ba44",
+                "LaunchTemplateName": "my-template-for-auto-scaling"
+            },
+            "MinSize": 1,
+            "MaxSize": 5,
+            ...
+            "Instances": [
+                {
+                    "InstanceId": "i-fc01551d496fc363f",
+                    "InstanceType": "t2.micro",
+                    "AvailabilityZone": "us-east-1a",
+                    ...
+                }
+            ],
+            ...
+            "TerminationPolicies": [
+                "Default"
+            ],
+            ...
+            "CapacityRebalance": false
+        }
+    ]
+}
+```
+
+### Attach an instance to the Auto Scaling group
+
+You can attach an instance to the Auto Scaling group using the [`AttachInstances`](https://docs.aws.amazon.com/autoscaling/ec2/APIReference/API_AttachInstances.html) API.
+
+Before that, create an EC2 instance using the [`RunInstances`](https://docs.aws.amazon.com/AWSEC2/latest/APIReference/API_RunInstances.html) API. Run the following commands to create an EC2 instance locally:
+
+{{< command >}}
+$ docker pull ubuntu:focal
+$ docker tag ubuntu:focal localstack-ec2/ubuntu-focal-docker-ami:ami-00a001
```

Comment:
This should not be necessary. The Pro provider will download the images at startup by default. You can replace the AMI ID with one of the supported ones from the LS EC2 docs